src/mixed-line-bar-chart/index.tsx

```diff
@@ -13,7 +13,7 @@ export { MixedLineBarChartProps };
 function MixedLineBarChart<T extends number | string | Date>({
   series = [],
   height = 500,
-  xScaleType = 'linear',
+  xScaleType = 'categorical',
```

Comment:
Would this not be a breaking change? I think we can get away with it in the Bar chart, because you can literally only use `categorical` there.

But you could theoretically build a Mixed line bar chart that only contains lines, right?

Reply:
No, I can use other scale types as long as I dont use bars. This is a Mixed Line Bar Chart:
![Screenshot 2023-05-25 at 08 31 13](https://github.com/cloudscape-design/components/assets/2446349/dc86f140-8fda-465d-a9d4-cb109ab8fb99)

Reply:
Right, I think I'll leave mixed charts unchanged.

Reply:
Changed mixed charts back to default to `linear` (even though it makes little sense to use mixed charts to display only lines?)